python/graphstorm/model/gat_encoder.py

```diff
@@ -134,16 +150,16 @@ class GATEncoder(GraphConvEncoder):
 
         # Build model and do full-graph inference on GATEncoder
         from graphstorm import get_node_feat_size
-        from graphstorm.model.gat_encoder import GATEncoder
```

Comment:
Does it work after changing the import source here?

Reply:
Yes. These layers and encoders have been imported in the `graphstorm/model/__init__.py` file. We can direct import them.